### ISSUE TITLE
Wrong react/jsx-indent since 6.7.0

### ISSUE THREAD

**Issue by ibc** [yannickcr/eslint-plugin-react#966]
Using `eslint-plugin-react` 6.2.1, eslint does not complain about the following JSX code:

```html
	{!state.localHold ?
		<PauseIcon
			onClick={this.handleHold.bind(this)}
		/>
	:
		<ResumeIcon
			onClick={this.handleResume.bind(this)}
		/>
	}
```

However, using 6.7.1, eslint complains:

```
93:9 error Expected indentation of 9 tab characters but found 8  (react/jsx-indent)
```

NOTE: line 93 is the one containing `<ResumeIcon`.

However, if I do the following, eslint does not complain:

```html
	{!state.localHold ?
		<PauseIcon
			onClick={this.handleHold.bind(this)}
		/>
	:
			<ResumeIcon
				onClick={this.handleResume.bind(this)}
			/>
	}
```

This is obviously weird.
